scripts/end_to_end_test/run_end_to_end_test.py

```diff
@@ -153,6 +154,9 @@ def append_node(self, index, load_directory=None):
             clear_path=False
         )
 
+        # Possibly soon to be depreciated functionality - set the block interval
+        instance._block_interval = self._block_interval
```

Comment:
Does this mean all test instances are now mining?

Reply:
no - instance._mining = True would set that